bc_obps/registration/tests/endpoints/v2/test_operators_endpoint.py

```diff
@@ -0,0 +1,104 @@
+from registration.constants import PAGE_SIZE
+from registration.tests.utils.helpers import CommonTestSetup, TestUtils
+
+from registration.tests.utils.bakers import operator_baker
+from registration.utils import custom_reverse_lazy
+
+
+class TestOperatorsEndpoint(CommonTestSetup):
+    endpoint = CommonTestSetup.base_endpoint + "v2/operators"
```

Comment:
We can update the endpoint here to `custom_reverse_lazy("list_operators_v2")` and then replace all occurrences of `custom_reverse_lazy("list_operators_v2")` in this file with `self.endpoint`. This way, if the endpoint name changes for any reason, we only need to update one variable.